casper/src/main/scala/coop/rchain/casper/Casper.scala

```diff
@@ -47,7 +49,7 @@ object DeployError {
   }
 }
 
-final case class BlockProcessingState(enqueued: Set[BlockHash], processing: Set[BlockHash])
+final case class BlockProcessingState(enqueued: Queue[BlockHash], processing: Set[BlockHash])
```

Comment:
I believe that if we decide to enable fair dispatcher back - this is is the queue on which it should be done.